Cesium3DTilesSelection/src/ImplicitOctreeLoader.cpp

```diff
@@ -221,10 +221,13 @@ ImplicitOctreeLoader::loadTileContent(const TileLoadInput& loadInput) {
             this->addSubtreeAvailability(
                 subtreeID,
                 std::move(*subtreeAvailability));
-          }
 
-          // tell client to retry later
-          return TileLoadResult::createRetryLaterResult(nullptr);
+            // tell client to retry later
+            return TileLoadResult::createRetryLaterResult(nullptr);
+          } else {
+            // Subtree load failed, so this tile fails, too.
+            return TileLoadResult::createFailedResult(nullptr);
+          }
```

Comment:
This change (and the similar one below) ensures that invalid subtrees are not loaded repeatedly, spamming an error to the log each time.